binding.gyp

```diff
@@ -45,19 +45,19 @@
         "-std=c++14"
       ],
       "variables": {
-        "nuodb_home": "<!(echo ${NUODB_HOME-\"/opt/nuodb\"})"
+        "nuodb_client_package": "<!(echo ${NUODB_CLIENT_PACKAGE:-${NUODB_HOME:-\"/opt/nuodb\"}})"
```

Comment:
The official environment variable that is defined to hold the root directory of the client package, like NUODB_HOME holds the root directory of the server package, is `NUOCLIENT_HOME` not `NUODB_CLIENT_PACKAGE`.